packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -844,6 +873,11 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
           * For new documents, we get this value based on the gcAllowed flag in runtimeOptions.
           */
         const prevSummaryGCVersion = existing ? getGCVersion(metadata) : undefined;
+
+        this.createDocRuntimeVersion = existing ? this.getCreateVersion(metadata) : pkgVersion;
+        this.createDocTimeStamp = existing ? this.getTimeStamp(metadata) : performance.now();
+        this.summaryCount = existing ? this.getSummaryCount(metadata) : 0;
```

Comment:
You don't need to initialize summaryCount from metadata. This information should be updated at every summary.